timon/plugins/http_server/views.py

```diff
@@ -73,10 +73,9 @@ async def get_queue():
     returns the probes in queue
     """
     heap = app.tmoncfg.get_queue().heap
-    items_in_heap = []
-    for item in heap:
-        items_in_heap.append(item)
-    return jsonify(items_in_heap)
+    copied_heap = heap.copy()
```

Comment:
there's not even a need to copy. Please read the docuimentation of the heap.

You can just sort the heap and jsonify it.

so
```
heap.sort()
return jsonify(heap)
```